src/transforms/dodge.d.ts

```diff
@@ -1,20 +1,56 @@
 import type {ChannelValueSpec} from "../channel.js";
 import type {Initialized} from "./basic.js";
 
-export type DodgeAnchor = "top" | "right" | "bottom" | "left" | "middle";
+export type DodgeAnchorX = "right" | "left" | "middle";
 
-export interface DodgeOptions {
-  x?: ChannelValueSpec;
+export type DodgeAnchorY = "top" | "bottom" | "middle";
+
+export interface DodgeOptionsX {
+  /** The vertical position of the mark. */
   y?: ChannelValueSpec;
+  /** The radius of the mark. */
   r?: ChannelValueSpec;
-  anchor?: DodgeAnchor;
+  /** The side of the frame used as an anchor. */
+  anchor?: DodgeAnchorX;
+  /** A number of pixels added to the radius of the mark. */
   padding?: number;
 }
 
-export function dodgeX<T>(options?: T & DodgeOptions): Initialized<T>;
-
-export function dodgeX<T>(dodgeOptions?: DodgeOptions | DodgeAnchor, options?: T): Initialized<T>;
+export interface DodgeOptionsY {
+  /** The horizontal position of the mark. */
+  x?: ChannelValueSpec;
+  /** The vertical position of the mark. */
+  y?: ChannelValueSpec;
```

Comment:
This is asymmetric with DodgeXOptions, which doesn’t allow _x_. I assume this is a mistaken copy-paste, so I will remove.